napari/utils/theme.py

```diff
@@ -220,12 +227,7 @@ def get_theme(name, as_dict=None):
         )
         as_dict = True
     if as_dict:
-        _theme = _theme.dict()
-        _theme = {
-            k: v if not isinstance(v, Color) else v.as_rgb()
-            for (k, v) in _theme.items()
-        }
-        return _theme
+        return _theme.dict()
```

Comment:
```suggestion
        return dict(_theme)
```

https://pydantic-docs.helpmanual.io/usage/exporting_models/#dictmodel-and-iteration